examples/README.md

```diff
@@ -2,52 +2,132 @@
 
 AMOP（Advanced Messages Onchain Protocol）即链上信使协议，旨在为联盟链提供一个安全高效的消息信道，联盟链中的各个机构，只要部署了区块链节点，无论是共识节点还是观察节点，均可使用AMOP进行通讯，AMOP有如下优势：
 
--   实时：AMOP消息不依赖区块链交易和共识，消息在节点间实时传输，延时在毫秒级。
--   可靠：AMOP消息传输时，自动寻找区块链网络中所有可行的链路进行通讯，只要收发双方至少有一个链路可用，消息就保证可达。
--   高效：AMOP消息结构简洁、处理逻辑高效，仅需少量cpu占用，能充分利用网络带宽。
--   安全：AMOP的所有通讯链路使用SSL加密，加密算法可配置,支持身份认证机制。
--   易用：使用AMOP时，无需在SDK做任何额外配置。
+- 实时：AMOP消息不依赖区块链交易和共识，消息在节点间实时传输，延时在毫秒级。
+- 可靠：AMOP消息传输时，自动寻找区块链网络中所有可行的链路进行通讯，只要收发双方至少有一个链路可用，消息就保证可达。
+- 高效：AMOP消息结构简洁、处理逻辑高效，仅需少量cpu占用，能充分利用网络带宽。
+- 安全：AMOP的所有通讯链路使用SSL加密，加密算法可配置,支持身份认证机制。
+- 易用：使用AMOP时，无需在SDK做任何额外配置。
 
 进一步了解 AMOP，请参考：[链上信使协议](https://fisco-bcos-documentation.readthedocs.io/zh_CN/latest/docs/manual/amop_protocol.html)
 
 **初始化**：
 
--   搭建单群组四节点区块链网络，可参考：[安装](https://fisco-bcos-documentation.readthedocs.io/zh_CN/latest/docs/installation.html)。
+- 搭建单群组四节点区块链网络，可参考：[安装](https://fisco-bcos-documentation.readthedocs.io/zh_CN/latest/docs/installation.html)。
 
 ## 单播案例
 
-**单播** 就是节点从监听同一 Topic 的多个订阅者中随机抽取一个订阅者转发消息
+**单播** 指的是节点从监听同一 Topic 的多个订阅者中随机抽取一个订阅者转发消息
 
--   启动 AMOP 服务端：
+- 运行 AMOP 消息发布者：go run examples/amop/unicast_pub/publisher.go [endpoint] [topic]
 
-    ```shell
-    go run examples/server/amop_server.go
-    ```
+  ```shell
+  > go run examples/amop/unicast_pub/publisher.go 127.0.0.1:20200 hello
+  
+    2020/08/10 22:43:31 publish message: hello, FISCO BCOS, I am unique broadcast publisher! 0 
+    2020/08/10 22:43:33 PushTopicDataRandom failed, err: sendMessage failed, err: error code 100, remote peer unavailable
+    2020/08/10 22:43:33 publish message: hello, FISCO BCOS, I am unique broadcast publisher! 1 
+    2020/08/10 22:43:35 PushTopicDataRandom failed, err: sendMessage failed, err: error code 100, remote peer unavailable
+    2020/08/10 22:43:35 publish message: hello, FISCO BCOS, I am unique broadcast publisher! 2 
+    2020/08/10 22:43:37 PushTopicDataRandom failed, err: sendMessage failed, err: error code 100, remote peer unavailable
+    2020/08/10 22:43:37 publish message: hello, FISCO BCOS, I am unique broadcast publisher! 3 
+    2020/08/10 22:43:39 publish message: hello, FISCO BCOS, I am unique broadcast publisher! 4
+  ```
 
-    控制台输出 "subscribe success" 表示服务启动成功
+- 启动 AMOP 消息订阅者：go run examples/amop/sub/subscriber.go [endpoint] [topic]
```

Comment:
Codacy found an issue: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/FISCO-BCOS/go-sdk/pullRequest?prid=5979849)